roles/common/files/sensu_plugins/check-cpu.rb

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/bin/env /opt/sensu/embedded/bin/ruby
 #
```

Comment:
These help us get system ruby free, but we may want to think about how we can support other rubies in the future.